pandas/io/excel/_base.py

```diff
@@ -755,6 +756,29 @@ class ExcelWriter(metaclass=abc.ABCMeta):
     >>> with ExcelWriter("path_to_file.xlsx", mode="a", engine="openpyxl") as writer:
     ...     df.to_excel(writer, sheet_name="Sheet3")
 
+    Here, the `if_sheet_exists` parameter can be set to replace a sheet if it
+    already exists:
+
+    >>> with ExcelWriter(
+    ...     "path_to_file.xlsx",
+    ...     mode="a",
+    ...     engine="openpyxl",
+    ...     if_sheet_exists="replace",
+    ... ) as writer:
+    >>>     df.to_excel(writer, sheet_name="Sheet1")
+
+    You can also write multiple DataFrames to a single sheet. Note that the
+    `if_sheet_exists` parameter needs to be set to `write_to` if you are in
+    append mode:
+
+    >>> with ExcelWriter("path_to_file.xlsx",
+    ...     mode="a",
+    ...     engine="openpyxl",
+    ...     if_sheet_exists="write_to",
+    ... ) as writer:
+    >>>     df.to_excel(writer, sheet_name="Sheet1")
+    >>>     df.to_excel(writer, sheet_name="Sheet1", startcol=3)
```

Comment:
It seems to me a more common use case would be to write two different frames to the same sheet; can you use df1 and df2 here.